CHANGELOG.md

```diff
@@ -4,6 +4,47 @@ All notable changes to Zebra will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-alpha.15](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-alpha.15) - 2021-08-TBD
+
+Zebra's latest alpha contains the last of the changes to zebra chain state ahead of NU5 testnet activation and before concentrating on mempool work. The remaining NU5 validation will be completed prior to NU5 mainnet activation.
+
+### Added
+
+#### Network Upgrade 5
+
+- ZIP-209: Prohibit Negative Shielded Chain Value Pool Balances Implementation (#2546, #2554, #2561, #2566, #2569, #2576, #2577, #2578, #2596)
+- ZIP-221: FlyClient - Consensus-Layer Changes Implementation(#2531, #2553, #2583)
```

Comment:
@conradoplg is the ZIP-221 implementation complete?
Or should we say that it's incomplete here?

```suggestion
- ZIP-221: FlyClient - Consensus-Layer Changes Implementation (#2531, #2553, #2583)
```

Reply:
There are still open ZIP-221 PRs, so I don't think this implementation is complete.

But I don't want to block on this, because we're only in alpha.

Reply:
I had to add #2547 to the list anyway so I clarified that this was only a partial implementation at the same time